test/e2e/e2e.go

```diff
@@ -1871,3 +1872,223 @@ func getNodePodCIDR(nodeName string) (string, error) {
+			}
+		}
+
+		//delete the db-pod in order to emulate the cluster/pod restart
+		framework.Logf("deleting db pod:%s", db_pod_name)
+		deletePod(f, ovnNs, db_pod_name)
+
+		//wait (let the new db pod to be ready)
+		framework.Logf("wait %v", waitingPeriod)
+		time.Sleep(waitingPeriod)
+
+		//check db files existence
+		checkDbFilesExistence(db_pod_num)
+	}
+	singlePodConnectivityTest := func(f *framework.Framework, podName string) {
+		framework.Logf("Running container which tries to connect to 8.8.8.8 in a loop")
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+
+		go checkContinuousConnectivity(f, "", podName, "8.8.8.8", 53, 30, podChan, errChan)
+		testPod := <-podChan
+
+		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
+		framework.ExpectNoError(<-errChan)
+	}
+	twoPodsConnectivityTest := func(f *framework.Framework,
+		node1Name string, node2Name string,
+		syncChan chan string, errChan chan error) {
+		const (
+			pod1Name                  string        = "connectivity-test-pod1"
+			pod2Name                  string        = "connectivity-test-pod2"
+			port                      string        = "8080"
+			timeIntervalBetweenChecks time.Duration = 2 * time.Second
+		)
+
+		var (
+			command = []string{"/agnhost", "netexec", fmt.Sprintf("--http-port=" + port)}
+		)
+		createGenericPod(f, pod1Name, node1Name, f.Namespace.Name, command)
+		createGenericPod(f, pod2Name, node2Name, f.Namespace.Name, command)
+
+		// wait for pods setup to return a valid address
+		wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
+			for _, podName := range []string{pod1Name, pod2Name} {
+				kubectlOut := getPodAddress(podName, f.Namespace.Name)
+				srcIP := net.ParseIP(kubectlOut)
+				if srcIP == nil {
+					return false, nil
+				}
+			}
+			return true, nil
+		})
+		pod2IP := getPodAddress(pod2Name, f.Namespace.Name)
+
+		ginkgo.By("Checking initial connectivity from one pod to the other and verifying that the connection is achieved")
+
+		stdout, err := framework.RunKubectl(f.Namespace.Name, "exec", pod1Name, "--", "curl", fmt.Sprintf("%s/hostname", net.JoinHostPort(pod2IP, port)))
+
+		if err != nil || stdout != pod2Name {
+			framework.Failf("Error: attempted connection to pod %s found err:  %v", pod2Name, err)
+		}
+
+		syncChan <- "connectivity test pods are ready"
+
+	L:
+		for {
+			select {
+			case msg := <-syncChan:
+				framework.Logf(msg + "finish connectivity test.")
+				break L
+			default:
+				stdout, err := framework.RunKubectl(f.Namespace.Name, "exec", pod1Name, "--", "curl", fmt.Sprintf("%s/hostname", net.JoinHostPort(pod2IP, port)))
+				if err != nil || stdout != pod2Name {
+					errChan <- err
+					framework.Failf("Error: attempted connection to pod %s found err:  %v", pod2Name, err)
+				}
+				time.Sleep(timeIntervalBetweenChecks)
+			}
+		}
+
+		errChan <- nil
+	}
+
+	table.DescribeTable("recovering from deleting db files and check simple connectivity from new pod to 8.8.8.8",
```

Comment:
To save time, you can test both continuous connectivity as well as new pod creation after deletion. In other words, combine the tests.

Reply:
done.